h4/h4.ino

```diff
@@ -854,6 +876,12 @@ void updateDisplay() {
     lcdHashLine2 = newHashLine2;
     charIndex = 0;
     lcd.setCursor(0, 2);
+
+    // Display current tool
```

Comment:
Only print in GCode mode and if tool isn't 0? Most people won't define their tools and should see no UI change.